src/App/ComponentsDocumentation/components/Forms/index.js

```diff
@@ -136,7 +143,7 @@ const Validation = () => (
                     id="validation-email-2"
                     type="email"
                     label="Email"
-                    placeholder="bob.corlsan@example.com"
```

Comment:
Why is poor bob corlsan being deleted? :'(

Reply:
Request from designers :(